socorro/scripts/pubsub.py

```diff
@@ -134,36 +134,19 @@ def status(config, args):
                 print('             %s' % crash_id)
 
 
-def publish_crashid(config, queue, crashids):
+def publish_crashid(config, args):
```

Comment:
I think I would have kept the arguments here as `queue` and `crashids` since it simplifies the setup for running this function--it's just working on a queue (str) and a list of crashids (list of crashids) rather than an argparse something or other.

I don't think we have to change this back. I don't think we test this function by itself.

Reply:
I prefer ``queue`` and ``crashids`` as well. It was ``args`` before I touched it, and I was matching the style of the other calls. I'm changing it back.

Reply:
Oh noes! I'm sorry for flip-flopping on you.